myapp/db/migrate/20250311025714_create_tasks.rb

```diff
@@ -0,0 +1,14 @@
+class CreateTasks < ActiveRecord::Migration[7.1]
+  def change
+    create_table :tasks do |t|
+      t.string :name
+      t.text :description
+      t.bigint :created_user_id
```

Comment:
I think it would be better to just name it `user_id` to make it easier to associate with the `User` model, which will probably be created later.
However, if the creator and the assignee are meant to be separate, I think the current naming is fine.